Sources/Purchasing/Purchases/PurchaseParams.swift

```diff
@@ -26,7 +26,7 @@ import Foundation
  *                            .with(metadata: ["key": "value"])
  *                            .with(promotionalOffer: promotionalOffer)
  *                            .build()
- *  Purchases.shared.purchase(package: package, params: params)
+ * Purchases.shared.purchase(params)
```

Comment:
looks good. Still not sure why we don't have these methods take in a Builder so that you don't have to remember to call `build()` yourself, that feels pretty pointless, but it's not related to this PR